opencv/src/main/scala/com/microsoft/azure/synapse/ml/opencv/ImageTransformer.scala

```diff
@@ -433,6 +433,9 @@ class ImageTransformer(val uid: String) extends Transformer
   def setStages(value: java.util.ArrayList[java.util.HashMap[String, Any]]): this.type =
     set(stages, value.asScala.toArray.map(_.asScala.toMap))
 
+  def setStages(value: Map[String, Any]): this.type =
```

Comment:
sparklyr does not support serialization/deserialization arrays of hash tables (lists of environments in R terms). And even if you call a scala method that produces a java hash map, by the time it gets back to R it's an R environment.  I plan to file a PR with sparklyr to add support for arrays of environments -- looks like a trivial fix. But for now, this works.

Reply:
Im not sure how one would set multiple stages with this API, is that supported in the R bindings?

Reply:
Ah, just got done removing all that json stuff.  But I can add it back here.